jekyll/_cci2/configuration-reference.md

```diff
@@ -406,25 +406,57 @@ A job that was not executed due to configured rules will show up in the list of
 
 **Note:** The `resource_class` feature is automatically enabled on Performance Plans. If you are on a container or unpaid plan you will need to [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) to have a CircleCI Sales representative contact you about enabling this feature on your account.
 
-It is possible to configure CPU and RAM resources for each job as described in the following table. If `resource_class` is not specified or an invalid class is specified, the default `resource_class: medium` will be used. The `resource_class` key is currently only available for use with the `docker` executor.
-
-Class       | vCPUs       | RAM
-------------|-----------|------
-small       | 1 | 2GB
-medium (default) | 2 | 4GB
-medium+     | 3 | 6GB
-large       | 4 | 8GB
-xlarge      | 8 | 16GB
-2XL         | 16 | 32GB
-2XL+        | 20 | 40GB
-1GPU        | 16 | 122GiB
-2GPU        | 32 | 244GiB
-4GPU        | 64 | 488GiB
+It is possible to configure CPU and RAM resources for each job. If `resource_class` is not specified, or an invalid class is specified, the default `resource_class: medium` will be used. Different resource classes are available for different executors, as described in the tables below. 
+
+##### Docker Executor
+
+Class            | vCPUs | RAM
+-----------------|-------|-----
+small            | 1     | 2GB
+medium (default) | 2     | 4GB
+medium+          | 3     | 6GB
+large            | 4     | 8GB
+xlarge\*         | 8     | 16GB
```

Comment:
xlarge doesn't require a support ticket

Reply:
@glenjamin Ok. Should I fix this box in this spreadsheet? Cell U9 indicates that xlarge requires approval. https://docs.google.com/spreadsheets/d/12siivROzyVbgHeOPJiKtC22wLucz-fUZPOVSGx9Ws54/edit#gid=821232712